packages/ember-data/lib/system/model/model.js

```diff
@@ -128,6 +131,22 @@ var Model = Ember.Object.extend(Ember.Evented, {
     Ember.deprecate('DS.Model#isDirty has been deprecated please use hasDirtyAttributes instead');
     return this.get('currentState.isDirty');
   }),
+
+  /**
+    @property error
+    @type {Boolean}
+    @deprecated
+  */
+  error: computedPolyfill({
+    get() {
+      if (!errorDeprecationShown) {
+        Ember.deprecate('DS.Model#error has been deprecated please use adapterError instead');
+        errorDeprecationShown = true;
+      }
+      return Ember.get(this, 'adapterError');
+    }
+  }),
```

Comment:
off cours... Sorry for the mess with this thing. Should have been an easy one. Updated...